frontend/app/electron/main/tray-manager.ts

```diff
@@ -40,6 +41,20 @@ export class TrayManager {
           ]
         : []),
       { type: 'separator' },
+      ...(this.config.isMac || this.config.isLinux
```

Comment:
So the `isLinux` is redundant here.

For the feature we update the [title](https://www.electronjs.org/docs/latest/api/tray#traysettitletitle-options-macos) which works only works for macOS.